Helper/RawConsoleReporter.php

```diff
@@ -45,7 +55,17 @@ public function onAfterRun(CheckInterface $check, ResultInterface $result, $chec
                 $this->output->write('FAIL');
         }
 
-        $this->output->writeln(sprintf(' %s', $check->getLabel()));
+        $performanceData = $this->getNagiosPerformanceData();
```

Comment:
RawConsolerReporter refers explicitly to Nagios with this function (it was already the case with the HealthCheckCommand's option "nagios" anyway), so I guess that a futur BC refactoring could rename the class accordingly, eg. "NagiosReporter".

All good to me, nice job keeping compatibility with previous code.